ribs.py

```diff
@@ -123,6 +123,16 @@ def clear_screen(color):
     bottom_right = pg.display.get_surface().get_size()
     pg.draw.rect(window, color, (top_left, bottom_right))
 
+
+def draw_text(text, position, size, color=pg.Color(255, 255, 255), font_name=None):
+
```

Comment:
A doc-string would be nice to have, and don't foreget to add it in the ribs.doc file aswell! :D